CHANGELOG.md

```diff
@@ -14,8 +15,51 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+## r1.3
+
+## Release Notes
+
+This public release contains the definition and documentation of
+* Customer Insights v0.1.1
+
+The API definition(s) are based on
+* Commonalities v0.5.0
+* Identity and Consent Management v0.3.0
+
+## Customer Insights v0.1.1
+
+**Customer Insights v0.1.1 is the second public release version of the Customer Insights API.**
+- **This version is a Patch Version for the former Public Release:**
+  - Fix a typo in an error example 
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/CustomerInsights/r1.3/code/API_definitions/customer-insights.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/CustomerInsights/r1.3/code/API_definitions/customer-insights.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/CustomerInsights/blob/r1.3/code/API_definitions/customer-insights.yaml)
+
+### Added
+* N/A
+
+### Changed
+* N/A
+
+### Fixed
+* Typo in error example `GENERIC_422_ID_DOCUMENT_NOT_SUPPORTED` status should read `422` in https://github.com/camaraproject/CustomerInsights/pull/28
+
+### Removed
+* N/A
+
+## New Contributors
+* N/A
+
+
+**Full Changelog**: https://github.com/camaraproject/CustomerInsights/compare/r1.1...r1.3
```

Comment:
In other patches, the full changelog is from the patched release

Reply:
"for a public release, the consolidated changes since the previous public release", so only the changes between r1.2 and r1.3, in result:
`**Full Changelog**: https://github.com/camaraproject/CustomerInsights/compare/r1.2...r1.3`

BTW: line 150 is wrong, it had to be `https://github.com/camaraproject/CustomerInsights/commits/r1.2` (all changes since the beginning, including the once of the release candidate)

Reply:
ok, i will apply in such a way

Reply:
@hdamker I guess you mean line 104

> BTW: line 150 is wrong, it had to be https://github.com/camaraproject/CustomerInsights/commits/r1.2 (all changes since the beginning, including the once of the release candidate)

Reply:
Ups, yes 104 ... from 150 i copied the template for the line.